run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+INITI_DATA= [ ! -f db.sqlite3 ]
+
+./manage.py makemigrations || exit 1
```

Comment:
just get rid of all these `|| exit 1`'s and use `set -eu` instead

Reply:
good point, will change it